build/lib/markdown.php

```diff
@@ -325,7 +325,7 @@ function transform($text) {
 		$text = $this->hashHTMLBlocks($text);
 
 		# Strip any lines consisting only of spaces and tabs.
-		# This makes subsequent regexen easier to write, because we can
+		# This makes subsequent regexes easier to write, because we can
```

Comment:
regexes is used throughout...

Reply:
I'd rather not have any changes made to the markdown.php file. It is a third party file, and I accept that there are some oddities in it, but for diffing externally I don't want any changes to make maintenance easier.